src/Symfony/Component/Notifier/Bridge/FreeMobile/FreeMobileTransport.php

```diff
@@ -26,7 +26,7 @@
  */
 final class FreeMobileTransport extends AbstractTransport
 {
-    protected const HOST = 'https://smsapi.free-mobile.fr/sendmsg';
+    protected const HOST = 'smsapi.free-mobile.fr/sendmsg';
```

Comment:
@fabpot 

Shall we throw an exception if `HOST` starts with `http` to avoid such bugs in the future or shall we ensure this by tests?

For reference:
https://github.com/symfony/symfony/blob/defb90b030b26143f677509e21f2bc59ea066be7/src/Symfony/Component/Notifier/Transport/AbstractTransport.php#L83-L86

Reply:
Tests are enough